libs/common-const/src/networks.ts

```diff
@@ -24,8 +23,6 @@ const DEFAULT_RPC_URL: Record<SupportedChainId, { url: string; usesInfura: boole
  */
 export const RPC_URLS: Record<SupportedChainId, string> = mapSupportedNetworks(getRpcUrl)
 
-export const MAINNET_PROVIDER = new JsonRpcProvider(RPC_URLS[SupportedChainId.MAINNET])
```

Comment:
<img width="1714" alt="Screenshot 2024-12-09 at 15 22 21" src="https://github.com/user-attachments/assets/aa7c1751-f22c-4d6f-bfd0-7bd0a3b3e434">

Reply:
This was not super obvious.
I noticed that some pages doesn't get cached.
Looking at the logs I found this error and started looking for a code who makes the infura call and eventually I got to `MAINNET_PROVIDER`.

So, `cow-fi` uses `@cowprotocol/common-consts` which exports `MAINNET_PROVIDER` which does `fetch('...infura')` call when create an instance of `JsonRpcProvider`.

Since this const is only used it `fetchTokenList.ts` I just moved it there.

Reply:
Next.js caches `fetch()` requests if you specify a corresponding parameter.
`JsonRpcProvider` calls `fetch()` inside with no cache parameter and it breaks next.js pages caching.